sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -943,3 +945,46 @@ def test_csr_row_norms(dtype):
     assert norms.dtype == dtype
     rtol = 1e-6 if dtype == np.float32 else 1e-7
     assert_allclose(norms, scipy_norms, rtol=rtol)
+
+
+@pytest.fixture(scope="module", params=CSR_CONTAINERS + CSC_CONTAINERS)
+def centered_matrices(request) -> tuple[sp.linalg.LinearOperator, np.ndarray]:
+    matrix_class = request.param
```

Comment:
I would rename this to `sparse_container_type` to move away from the "matrix" naming scheme.

Reply:
```suggestion
    """Two equivalent centered matrices as a LinearOperator and dense np.ndarray.
    """
    sparse_container_type = request.param
```

Reply:
I went with `sparse_container` since that's what I see most tests that use the `CSR_CONTAINER`, `CSC_CONTAINER` variables do.